assets/src/scss/partials/navigation-bar-light.scss

```diff
@@ -15,14 +15,5 @@
 
 #nav-mobile-menu {
   --nav-link--color: #{$grey-40};
-  --nav-link--active--color: #{$black};
```

Comment:
I think that this was not needed since we already set it to `black` earlier on in the file